src/store/actions/session.js

```diff
@@ -22,9 +24,22 @@ import { getUserProfile } from './user'
  * @returns {Function} Redux action thunk
  */
 export const logout = (ctx) => {
-  return (dispatch, getState) => {
+  return async (dispatch, getState) => {
     deleteCookie('access_token', ctx)
 
+    const token = selectSessionToken(getState())
+
+    await dispatch(createAxiosFSA(
```

Comment:
so I know I did this, but awaiting this without any UI feedback is probably bound to cause some confusion. I think it's easiest to just remove the await and do this step in the background without the user knowing it.